app/models/existing_role.rb

```diff
@@ -0,0 +1,34 @@
+class ExistingRole < ActiveRecord::Base
```

Comment:
Why is this model called `ExistingRole` instead of something as `UserRole` or just `Role`?